src/layers/CasaLayer/CasaLayer.js

```diff
@@ -57,6 +59,13 @@ import ArrowImg from '../../img/arrow.png';
  * @param {styleFunction} [options.styleFunction] Style function.
  */
 class CasaLayer extends VectorLayer {
+  static isRoute = (geometry) => {
```

Comment:
Please do the route styling in the route layer and not inside the generic `CasaLayer`. The route styling doesn't have to be generic. Moreover, for readability I would not use an extra function for this simple `if` condition.

Reply:
I did the styling in  CASA Layer, since all the other styles are defined here as well, including route styles (like the arrows for instance). I will move all the styling into route/zone layers respectively.

Reply:
I created the isRoute function to actually improve readability, since the if condition would be quite long and is used multiple times:
geometry instanceof LineString || geometry instanceof MultiLineString

Reply:
The generic styling functionality is done in the casa layer. Route styling should be done in the route layer.